### ISSUE TITLE
Shader variable hints and defaults don't work together

### ISSUE THREAD

**Issue by LikeLakers2** [godotengine/godot#17907]
**Godot version:**
Godot Engine v3.0.2.stable.official

**OS/device including version:**
Windows 10 Home v1709 (Fall Creator's Update); Nvidia GTX 660

**Issue description:**
I am unable to mix variable hints and defaults within shaders. If I attempt to, Godot tells me there's an error with my syntax.

Here's what I tried to make it work, with neither working.
```shader
uniform vec4 color : hint_color = vec4(0.8, 0.0, 0.0, 1.0);
// SCRIPT ERROR: (null): Expected ';'
// ...despite the ';' at the end?
```
```shader
uniform vec4 color = vec4(0.8, 0.0, 0.0, 1.0) : hint_color;
// SCRIPT ERROR: (null): Expected expression, found: CURSOR
```

Either my syntax is somehow wrong, or the two just don't work together (and the docs say nothing about this situation, so...).

**Steps to reproduce:**
Attempt to mix variable hints and defaults as shown above.

**Minimal reproduction project:**
N/A

**Comment by JFonS** [comment]
This has already been fixed with #17533 and should be available in the next release.

**Comment by LikeLakers2** [comment]
Oh, alright! I must of missed that issue during my search before submitting this.
